### ISSUE TITLE
Adding Sinhala

### ISSUE THREAD

**Issue by pathumego** [impallari/Font-Testing-Page#46]
@impallari I have done a sinhala iteration of tester. Can we add it to main tester?

**Comment by impallari** [comment]
Yes! We can do it after Typoday (I'm flying to Mumbai tomorrow.)

**Comment by pathumego** [comment]
Ah, cool.. Ill meet you there :)

**Pathum Egodawatta*

*Graphics Design B.A. (Hons)*

*twitter <https://twitter.com/pathumego>  **Portfolio
<http://iampathum.com/>*
